continuationsPlugin/src/main/scala/continuations/ContinuationsPlugin.scala

```diff
@@ -48,6 +50,26 @@ class ContinuationsPhase extends PluginPhase:
     report.logWith("transformBlock")(tree.show)
     tree
 
+  var continuationTraitSym: ClassSymbol = _
+  var continuationObjectSym: TermSymbol = _
+  var safeContinuationClassApplySym: TermSymbol = _
+  var interceptedMethodSym: TermSymbol = _
+
+  override def prepareForUnit(tree: Tree)(using Context): Context =
+    continuationTraitSym = requiredClass("continuations.Continuation")
+    continuationObjectSym = requiredModule("continuations.Continuation")
+
+    safeContinuationClassApplySym = requiredModule("continuations.SafeContinuation")
+      .requiredMethod("apply", List(defn.AnyType, defn.AnyType))
+
+    interceptedMethodSym =
+      requiredPackage("continuations.intrinsics").requiredMethod("intercepted")
+
+    ctx
```

Comment:
❓ I'm not an expert on the dotty compiler (yet), but is `prepareForUnit` where we want to do this work? Wouldn't we want to do this in `prepareForDefDef` since we're transforming a `DefDef` tree?

Reply:
yep you are right https://github.com/47deg/TBD/pull/52/commits/1a5f9ef8011ed082a480c0da6afa4bae1cc71944#diff-bfdffcfa12331080aaef50b40eae8ba34ac1bbfa58bb69c99d4ceef4b5607d7cR55